### ISSUE TITLE
VST3 Output Bus channelCount set to wrong value

### ISSUE THREAD

**Issue by PolyVector** [AuburnSounds/Dplug#682]
Hiya,
I was having an issue with Ardour not giving my VST3 instrument any outputs.  Tracked it down to a small copy/paste error where the Output channelCount is set to the maxInputs.

_vst3/dplug/vst3/client.d on Line 143_
```
            with(busAudioOut.info)
            {
                mediaType = kAudio;
                direction = kOutput;
                channelCount = maxInputs; /* ### This should be maxOutputs ### */
                setName("Audio Output"w);
                busType = kMain;
                flags = BusInfo.BusFlags.kDefaultActive;
            }
```

Oddly, most other hosts appear to work despite my plugin reporting zero outputs.

**Comment by p0nce** [comment]
Thanks for the discovery. Probably this wasn't catched before because we would release mostly effects in 1-1 and 2-2 configurations.

Anyway this is fixed in v12.7.13 you just have to `dub upgrade`

**Comment by p0nce** [comment]
Note: need to prioritize #657 more because of all the people making instruments.
